resources/[system]/scoreboard/ui/App.css

```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600&display=swap');
+
+* {
+  font-family: 'Rubik', sans-serif;
+}
+
+.scoreboardWrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+.scoreboard {
+  background-color: #414a4d;
+  width: 1000px;
+  max-width: 100%;
+  height: 600px;
+  overflow: auto;
+  background-image: url("nui://scoreboard/assets/backdrop.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.content-table {
+  border-collapse: separate;
+  width: 100%;
+  table-layout: fixed;
+  border: none;
+  margin: 0;
+  border-spacing: 0;
+}
+
+.content-table thead th {
+  position: sticky;
+  top: 0;
+  z-index: 1000;
+  background: rgba(35, 35, 35, 1);
+  border-bottom: 2px solid rgba(244,5,82,0.75);
+  text-align: left;
+  color: #fff;
+  font-weight: bold;
+}
+
+.content-table th,
+.content-table td {
+  padding: 12px 12px;
+  min-width: 50px;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+.content-table tbody tr {
+  color: #fff;
+  transition: .2s;
+  border-bottom: 1px solid #eeeeee;
+  background: rgba(35, 35, 35, 0.6);
```

Comment:
This'll lead to the background image being unevenly displayed. Opacity should be the same for both and color should be pre-blended in that case.